fpdf/template.py

```diff
@@ -74,26 +74,37 @@ def load_elements(self, elements):
         self.elements = elements
         self.keys = [v["name"].lower() for v in self.elements]
 
+    def _parse_colorcode(self, s):
+        """ Allow hex and oct values for colors """
+        s = s.strip()
+        if not s:
+            raise ValueError('Foreground and Background must be numeric')
+        if s[:2] in ['0x', '0X']:
+            return int(s, 16)
+        elif s[0] == '0':
+            return int(s, 8)
+        return int(s)
+
     def parse_csv(self, infile, delimiter=",", decimal_sep=".", encoding=None):
         """Parse template format csv file and create elements dict"""
-        keys = (
-            "name",
-            "type",
-            "x1",
-            "y1",
-            "x2",
-            "y2",
-            "font",
-            "size",
-            "bold",
-            "italic",
-            "underline",
-            "foreground",
-            "background",
-            "align",
-            "text",
-            "priority",
-            "multiline",
+        handlers = (
+            ("name", str.strip),
+            ("type", str.strip),
+            ("x1", float),
```

Comment:
i like this approach. "in the name of deterministic behavior" why strip the fields? but this is a good place to put the actual conversions, and less lines of code than what i was considering when i put the util function in.

Reply:
Everything got stripped in the original version, so I just left it that way in order not to change the behaviour. The alternative would be to just use `str` as a NOP.